PWGDQ/DataModel/ReducedInfoTables.h

```diff
@@ -284,6 +284,17 @@ DECLARE_SOA_TABLE(ReducedMuonsLabels, "AOD", "RTMUONSLABELS", //!
 
 using ReducedMuonsLabel = ReducedMuonsLabels::iterator;
 
+namespace dilepton_track_index
+{
+DECLARE_SOA_INDEX_COLUMN_FULL(Index0, index0, int, Tracks, "_0"); //! Index to first prong
+DECLARE_SOA_INDEX_COLUMN_FULL(Index1, index1, int, Tracks, "_1"); //! Index to second prong
+} // namespace dilepton_track_index
+
+DECLARE_SOA_TABLE(Dq2Prong, "AOD", "DQ2PRONG", //! Table for HF 2 prong candidates
+                  o2::soa::Index<>,
+                  dilepton_track_index::Index0Id,
+                  dilepton_track_index::Index1Id);
```

Comment:
I don't understand why we need this table. I don't see it being used anywhere. If indeed not used, please remove it and we can discuss if there is a case for having this in.